From 58195c195110f0dbb24d831b5eefb2f094848b26 Mon Sep 17 00:00:00 2001
From: Christian Ruppert <idl0r@qasl.de>
Date: Sun, 03 Nov 2013 17:38:55 +0000
Subject: Require alsa only with --enable-client

Hey guys,

the alsa libraries/header seem to be required only when configured with
--enable-client so I wrote a patch to make configure respect it.

--
Regards,
Christian Ruppert

From 53683cc75ba092799f856f710cb45b2aacfb6123 Mon Sep 17 00:00:00 2001
From: Christian Ruppert <idl0r@gentoo.org>
Date: Sun, 3 Nov 2013 18:36:26 +0100
Subject: [PATCH] Require alsa only with --enable-client
---
diff --git a/configure.ac b/configure.ac
index 7e81329..882bf1b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -246,12 +246,6 @@ fi
 AC_MSG_RESULT([found])
 fi
 
-if test "$os_linux" = yes; then
-	PKG_CHECK_MODULES(ALSA, alsa)
-	AC_SUBST(ALSA_CFLAGS)
-	AC_SUBST(ALSA_LIBS)
-fi
-
 PKG_CHECK_MODULES(SSL, openssl)
 AC_SUBST(SSL_CFLAGS)
 AC_SUBST(SSL_LIBS)
@@ -280,6 +274,12 @@ AC_SUBST(GL_LIBS)
 SPICE_NONPKGCONFIG_LIBS+=" $GL_LIBS"
 
 if test "x$red_target" = "xx11" && test "x$enable_client" = "xyes" ; then
+	if test "$os_linux" = yes; then
+		PKG_CHECK_MODULES(ALSA, alsa)
+		AC_SUBST(ALSA_CFLAGS)
+		AC_SUBST(ALSA_LIBS)
+	fi
+
 	PKG_CHECK_MODULES(XRANDR, xrandr)
 	PKG_CHECK_MODULES(XFIXES, xfixes)
 	PKG_CHECK_MODULES(MISC_X, x11 xext xrender)
--
cgit v0.9.0.2-2-gbebe
